### DISCUSSION TITLE
Frequency increment

### DISCUSSION THREAD

**Discussion by sarriegi** [NOAA-EMC/WW3#875] [Q&A]
Hello, if I set the first two numbers of the first line in ww3_grid.inp to 1.1 and 0.05, then when I look at the frequencies in the output that comes from the ww3_ounp file, then indeed the first frequency is 0.05. But the increment is not 1.1. The second frequency is 0.055, the third is 0.0605, the fourth is 0.0666 and in general the increment is not constant. How could I set a constant increment of let's say 0.01?
@aliabdolali 

Angelos

**Comment by aliabdolali** [comment]
The freq is in log scale. If increment us 1.1 and first frequency is f0, then f=f0, 1.1f0, 1.1^2f0, 1.1^3f0,...

**Reply by aliabdolali** [reply]
And it is like this in the outputs, some terms in the model are also calibrated with either 1.1 or 1.07. You can try different increment, not constant but i would recommend not trying other increment than 1.1 or 1.07

**Reply by sarriegi** [reply]
I understand, thank you!

Angelos
